alfresco-transform-core-aio/alfresco-transform-core-aio-boot/pom.xml

```diff
@@ -137,13 +137,23 @@
                     <plugin>
                         <groupId>io.fabric8</groupId>
                         <artifactId>docker-maven-plugin</artifactId>
+                        <version>0.36.1</version>
+                        <extensions>true</extensions>
                         <configuration>
+                            <apiVersion>1.29</apiVersion>
+                            <imagePullPolicy>Always</imagePullPolicy>
+                            <autoCreateCustomNetworks>true</autoCreateCustomNetworks>
+                            <!--<dockerHost>tcp://127.0.0.1:2376</dockerHost>-->
                             <images>
                                 <image>
                                     <alias>activemq</alias>
                                     <name>alfresco/alfresco-activemq:5.16.1</name>
                                     <run>
                                         <hostname>activemq</hostname>
+                                        <network>
+                                            <name>transform</name>
+                                            <alias>activemq</alias>
+                                        </network>
```

Comment:
You'll likely also need to add this to the aio image below and I don't think the alias is needed.